test/plugins/scenarios/rocketmq/main.go

```diff
@@ -0,0 +1,246 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/apache/rocketmq-client-go/v2"
+	"github.com/apache/rocketmq-client-go/v2/consumer"
+	"github.com/apache/rocketmq-client-go/v2/primitive"
+	"github.com/apache/rocketmq-client-go/v2/producer"
+
+	_ "github.com/apache/skywalking-go"
+)
+
+type testFunc func() error
+
+const (
+	uri   = "http://mqnamesrv:9876"
+	retry = 2
+	topic = "sw-topic"
+	group = "sw-group"
+	msg   = "I love skywalking %s thousand"
+)
+
+func main() {
+	route := http.NewServeMux()
+	route.HandleFunc("/execute", func(res http.ResponseWriter, req *http.Request) {
+		_, _ = res.Write([]byte("execute success"))
+	})
+	route.HandleFunc("/health", func(writer http.ResponseWriter, request *http.Request) {
+		_, _ = writer.Write([]byte("ok"))
+	})
+
+	TestProCon()
```

Comment:
Please invoke this method in the execute handler, it more like the real scene.

Reply:
Because I encountered the same problem before during development https://github.com/apache/skywalking-go/pull/165#issuecomment-1935516907

Reply:
I mean, you should adding send message(producer) to the http handler, the consumer should other thing. Because you have two entry span, so the span name been rewrited.

Reply:
very sorry, I have been stuck on this question for a long time, based on Teacher Wu and your suggestions, I have tried it like this, the UI is displayed as follows

<img width="400" alt="image" src="https://github.com/apache/skywalking-go/assets/66550292/44a7a18e-7025-4a66-a7e5-d3855e03df1f">

[excepted.yml](test/plugins/scenarios/rocketmq/config/excepted.yml), If there are still problems, please prompt me with more info, thanks

Reply:
![image](https://github.com/apache/skywalking-go/assets/3417650/bc23ccce-4458-48ee-9eef-1ed56fa81101)
They have same span name, I think I could make user confuse, should we change names?

Reply:
ok, I will use different operationName to identify async and its corresponding callback.